content/image.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+
+const { ROOTS } = require("./constants");
+const { memoize, slugToFolder } = require("./utils");
+
+function urlToFilePath(url) {
+  const [, locale, , ...slugParts] = url.split("/");
+  return path.join(locale.toLowerCase(), slugToFolder(slugParts.join("/")));
+}
+
+const find = memoize((relpath) => {
+  const filePath = ROOTS.map((root) =>
+    path.join(root, relpath)
+  ).find((filePath) => fs.existsSync(filePath));
```

Comment:
`find` returns `undefined` if nothing is found, so let's just return it and skip the unnecessary check below